prov/verbs/src/ep_rdm/verbs_tagged_ep_rdm.c

```diff
@@ -87,16 +87,25 @@ fi_ibv_rdm_prepare_rma_request(struct fi_ibv_rdm_request *request,
 
 static int fi_ibv_rdm_tagged_getname(fid_t fid, void *addr, size_t * addrlen)
 {
-	struct fi_ibv_rdm_ep *ep =
-		container_of(fid, struct fi_ibv_rdm_ep, ep_fid);
+	struct fi_ibv_rdm_ep *ep;
+
+	if (fid->fclass == FI_CLASS_EP)
+ 		ep = container_of(fid, struct fi_ibv_rdm_ep, ep_fid);
+	else {
```

Comment:
nit: Ok, this is the last one. :) We need braces here since the else has multiple statements.
reference: https://github.com/torvalds/linux/blob/master/Documentation/process/coding-style.rst